frontend/routes/components/topMenu/components/visualSettingsDialog/visualSettingsDialog.styles.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ *  Copyright (C) 2019 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+import { COLOR } from '../../../../../styles';
+import { Button, DialogContent, ListItem, Tooltip, Input, Tabs, Tab } from '@material-ui/core';
+import * as React from 'react';
+import { TooltipProps } from '@material-ui/core/Tooltip';
+import {omit} from 'lodash';
+
+export const NegativeActionButton = styled(Button)`
+  && {
+    color: ${COLOR.WHITE_87};
+	background-color:  rgba(234, 57, 57, 0.87);
```

Comment:
Could you move repeatable colors to shared colors or to some constant in this file?